### ISSUE TITLE
add new package: librsb

### ISSUE THREAD

**Issue by michelemartone** [spack/spack#14454]
Hi. Maybe librsb would be a good add to Spack.

**Comment by Sinan81** [comment]
it seem it only depends libc and libgomp which are provided by OS and gcc respectively. Hence, adding the package is just a matter of doing:
```
spack create https://sourceforge.net/projects/librsb/files/librsb-1.2.0.8.tar.gz/download
```
Have you tried that yet?

**Comment by michelemartone** [comment]
On 20200111@23:58, Sinan wrote:
> it seem it only depends libc and libgomp which are provided by OS and gcc respectively. Hence, adding the package is just a matter of doing:
> ```
> spack create https://sourceforge.net/projects/librsb/files/librsb-1.2.0.8.tar.gz/download
> ```
> Have you tried that yet?
> 
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/spack/spack/issues/14454#issuecomment-573391600

Trying that produces a package template with commented extra possible
lines, which I understand are optional, right ?

**Comment by adamjstewart** [comment]
Closed by #14455
